pkgs/tools/networking/i2p/default.nix

```diff
@@ -13,11 +13,15 @@
 
 stdenv.mkDerivation rec {
```

Comment:
Could you switch this over to using `stdenv.mkDerivation (finalAttrs: {` instead of `rec`? (Introduced in <https://github.com/NixOS/nixpkgs/pull/119942>.)

Reply:
Done.